galaxy_ng/app/tasks/publishing.py

```diff
@@ -94,3 +104,11 @@ def import_and_auto_approve(temp_file_pk, **kwargs):
         log.info('Imported and auto approved collection artifact %s to repository %s',
                  collection_version.relative_path,
                  golden_repo.latest_version())
+
+        if settings.GALAXY_ENABLE_API_ACCESS_LOG:
```

Comment:
are there any unit or functional tests for api logging where this msg could be added?